test/k8sT/Updates.go

```diff
@@ -143,10 +143,11 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldHelmChartVers
 		_ = kubectl.ExecMiddle("helm delete cilium --namespace=" + helpers.CiliumNamespace)
 		_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete configmap --namespace=%s cilium-config", helpers.CiliumNamespace))
 		_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete serviceaccount --namespace=%s cilium cilium-operator", helpers.CiliumNamespace))
-		_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete clusterrole --namespace=%s cilium cilium-operator", helpers.CiliumNamespace))
-		_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete clusterrolebinding --namespace=%s cilium cilium-operator", helpers.CiliumNamespace))
+		_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete clusterrole --namespace=%s cilium cilium-operator cilium-psp cilium-operator-psp", helpers.CiliumNamespace))
+		_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete clusterrolebinding --namespace=%s cilium cilium-operator cilium-psp cilium-operator-psp", helpers.CiliumNamespace))
 		_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete daemonset --namespace=%s cilium", helpers.CiliumNamespace))
 		_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete deployment --namespace=%s cilium-operator", helpers.CiliumNamespace))
+		_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete podsecuritypolicy --namespace=%s cilium-psp cilium-operator-psp", helpers.CiliumNamespace))
```

Comment:
please replace this line with `_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete podsecuritypolicy cilium-psp cilium-operator-psp"))` since psp are not namespaced